bot/cogs/cyber.py

```diff
@@ -368,7 +370,16 @@ def suffix_from_number(num):
             text += "We’re currently allocating students to their preferred locations so it’s an ongoing process!"
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
-
-
+        #Level Function Regex.
+        match = self.game_level_regex.match(message.content)
+        if match:
+            level_num, challenge_num = int(match.group(2)), int(match.group(3))
+            if match and message.channel.id == '532654287386443777':
+                base = 'hq'
+            elif match and message.channel.id == '532654358660120586':
+                base = 'm'
+            elif match and message.channel.id == '32654399453921291':
```

Comment:
These channel IDs should ideally be placed in the constants file (src/bon/constants.py)